frontend/src/components/RecipeCard/RecipeCard.jsx

```diff
@@ -1,6 +1,7 @@
 import { Card } from './RecipeCard.styles.js';
 
 
+
 const RecipeCard = ({ recipe }) => {
   console.log('this is recipe', recipe)
   console.log('this is image', recipe.image)
```

Comment:
Can you take the console.logs out for the PR? You can keep them in your local branch if they are helpful for debugging but I think its best to not push the console stuff out to everyone.